### ISSUE TITLE
Pretranslate with strings from git

### ISSUE THREAD

**Issue by jnavila** [jnavila/git-manpages-l10n#22]
Some parts of git inline help are very close to the documentation strings. This task could be applied to all available languages in git and could help kickstart translations in other languages.

**Comment by jnavila** [comment]
"translate toolkit" provides a utility name `pretranslate` that do the job.

**Comment by rimrul** [comment]
I've just done a quick test-run with `po/documentation.de.po` from current master and `de.po` from git/git's current master. The command I ran was 

`pretranslate po/documentation.de.po -t de.po -o po/documentation.de.po`.

It works, but needs manual review (a carefull `git add -p`, probably). It marks all strings, even exact matches and unchanged strigs as fuzzy.

**Comment by jnavila** [comment]
OK; so this is at most a one-time process. I haven't tested for French. Do you think it's worth it?

**Comment by rimrul** [comment]
Yes, but I'm still thinking of a way to filter out the changes that are just marking an existing string as fuzzy.

Also it's probably more of a once every couple git releases thing.

**Comment by jnavila** [comment]
I used the following line: 

```shell
$ pretranslate -t po/documentation.fr.po -i po/documentation.pot --tm=../git/po/fr.po -o po/documentation.pre.fr.po
```
In fact, I think that's exactly the way it has to be used for new versions of po/documentation.pot

**Comment by jnavila** [comment]
I did it for es_MX and ru.
